src/cascadia/TerminalApp/TerminalPage.cpp

```diff
@@ -805,12 +805,12 @@ namespace winrt::TerminalApp::implementation
     //      currently displayed, it will be shown.
     // Arguments:
     // - settings: the TerminalSettings object to use to create the TerminalControl with.
-    void TerminalPage::_CreateNewTabFromSettings(GUID profileGuid, TerminalSettings settings)
+    void TerminalPage::_CreateNewTabFromSettings(GUID profileGuid, Microsoft::Terminal::Settings::Model::TerminalSettingsStruct settings)
```

Comment:
while you are changing the signature here, can you make this `const TerminalSettingsThing&`?